src/tools/dartpad.md

```diff
@@ -4,21 +4,29 @@ description: The tool that lets you interactively play with Dart in a browser.
 permalink: /tools/dartpad
 ---
 
-DartPad, an open-source tool,
-allows you to play with the Dart language features in any modern browser.
+<a href="{{site.custom.dartpad.direct-link}}"
+target="_blank">DartPad (dartpad.dartlang.org)</a>
+is an open-source tool that
+lets you play with the Dart language in any modern browser.
+Here's what DartPad looks like:
 
-As you can see in the following screenshot, DartPad launches with a simple counting
-function it's been pre-loaded with a different example:
+<img src="images/DartPadWindow.png" alt="DartPad screenshot" />
 
-<img src="images/DartPadWindow.png" alt="a screenshot showing what DartPad looks like" />
 
-It's easy to get started!
+## Getting started
 
-## Open DartPad, and run some samples {#step-1-open-and-run}
+To get familiar with DartPad, try these steps:
+
+* [Run some samples.](#step-1-open-and-run)
+* [Create a simple command-line app.](#step-2-server)
+* [Create a simple web app.](#step-3-web)
+
+
+### Open DartPad, and run some samples {#step-1-open-and-run}
 
 <ol markdown="1">
```

Comment:
Eventually, these should be converted to markdown lists (something I started doing in the ACX Lottery tutorial and elsewhere).